README.md

```diff
@@ -43,22 +96,35 @@ cd SmartSim-Scaling
 pip install -r requirements.txt
 ```
 
-You will need to install ``mpi4py`` in your python environment. The install instructions
-can be found by selecting [mpi4py docs](https://mpi4py.readthedocs.io/en/stable/install.html).
+Lastly, the C++ applications themselves need to be built. To complete this, 
+one CMake edit is required. Navigate to your respective tests `CMakeLists.txt` file.
+If you need assistance finding the file path, please reference the chart below.
+Near the top of `CMakeLists.txt`, on line
+`set(SMARTREDIS "../../SmartRedis" CACHE PATH "Path to SmartRedis root directory")`, 
+change the path to the ``SMARTREDIS`` variable to
+the top level of the directory where you built or installed the SmartRedis library. 
```

Comment:
I think the user should rather define `SMARTREDIS` when the they issue the cmake command e.g.

```
cmake .. -DSMARTREDIS=/path/to/SmartRedis
```